core/src/main/java/org/opensearch/sql/expression/function/FunctionProperties.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.expression.function;
+
+import java.io.Serializable;
+import java.time.Clock;
+
+public interface FunctionProperties extends Serializable {
```

Comment:
does the interface required? i think NONE could extend FunctionProperties, and QueryFunctionProperties could be removed.

Reply:
Done in ea4588e3ca86157b709bd7184a27edc79253e8f5